src/utils/legacyLayerParamUtils.js

```diff
@@ -10,7 +10,7 @@ function readUrlParamValue(url, paramName) {
     return undefined
 }
 
-const newLayerParamRegex = /^[\w\\.]+[@\w=]*[,ft]*[,?\d\\.]*$/
+const newLayerParamRegex = /^[\w.]+[@\w=]*[,ft]*[,?\d.]*$/
```

Comment:
If layer ID never contains numbers, then it fines otherwise you might want to do `/^[\w\d.]+[@\w=]*[,ft]*[,?\d.]*$/`